src/lib/commands/remoteFs.ts

```diff
@@ -0,0 +1,28 @@
+import { ExtensionContext, window, commands } from 'vscode';
+import { COMMANDS } from '../constants';
+const { deleteFile } = require('@hubspot/cli-lib/api/fileMapper');
+const { getPortalId } = require('@hubspot/cli-lib');
+
+export const registerCommands = (context: ExtensionContext) => {
+  context.subscriptions.push(
+    commands.registerCommand(
+      COMMANDS.REMOTE_FS.DELETE,
+      async (clickedFileLink) => {
+        console.log(COMMANDS.REMOTE_FS.DELETE);
+        const fileLinkIsFolder = clickedFileLink.icon === 'symbol-folder';
+        const filePath = fileLinkIsFolder
+          ? clickedFileLink.path
+          : clickedFileLink.url.slice(5);
+        const selection = await window.showWarningMessage(
+          `Are you sure you want to delete ${filePath}? This action cannot be undone.`,
+          ...['Okay', 'Cancel']
```

Comment:
Just poked around in the app to see what other tools have for the confirmation content (was mainly curious but will toss it down below if helpful at all). Some other examples include:
- Delete this file/folder (when deleting something from design manager)
- Delete (when deleting a page)
- Yes, delete (when deleting a blog)

Reply:
Gotcha! We have the `hs remove` command as well in the CLI that has no warning beforehand so I think this is probably fine enough warning.